src/miral/input_device_config.cpp

```diff
@@ -42,15 +42,21 @@ auto get_optional(std::shared_ptr<mir::options::Option> options, char const* nam
 };
 
 char const* const disable_while_typing_opt = "touchpad-disable-while-typing";
-char const* const tap_to_click_opt = "touchpad-tap-to-click";
-char const* const mouse_acceleration_opt = "mouse-acceleration";
+char const* const disable_with_external_mouse_opt = "touchpad-disable-with-external-mouse";
+char const* const touchpad_tap_to_click_opt = "touchpad-tap-to-click";
+char const* const mouse_handedness_opt = "mouse-handedness";
+char const* const right = "right";
+char const* const left = "right";
+char const* const mouse_cursor_acceleration_opt = "mouse-cursor-acceleration";
 char const* const acceleration_none = "none";
 char const* const acceleration_adaptive = "adaptive";
 char const* const mouse_cursor_acceleration_bias_opt = "mouse-cursor-acceleration-bias";
-char const* const mouse_scroll_speed_scale_opt = "mouse-scroll-speed-scale";
+char const* const mouse_vscroll_speed_opt = "mouse-scroll-vspeed";
+char const* const mouse_hscroll_speed_opt = "mouse-scroll-hspeed";
 char const* const touchpad_cursor_acceleration_opt = "touchpad-cursor-acceleration";
 char const* const touchpad_cursor_acceleration_bias_opt = "touchpad-cursor-acceleration-bias";
-char const* const touchpad_scroll_speed_scale_opt = "touchpad-scroll-speed-scale";
+char const* const touchpad_vscroll_speed_opt = "touchpad-vscroll-speed";
+char const* const touchpad_hscroll_speed_opt = "touchpad-hscroll-speed";
```

Comment:
Oh no, don't like that… 99% just need the same scale for both h/v, I'd rather stick to a single float, or maybe:
```
--mouse-scroll-speed {<hvscale>|<hscale>,<vscale>}
```

Reply:
Maybe: use `--mouse-scroll-speed` for both unless `--mouse-horizontal-scroll-speed-override` is also set?

Reply:
But then… what if you only need vscroll changed? I suppose my proposal also forced you to give both, but in a more obvious way…

Reply:
Maybe: use `--mouse-scroll-speed` for both unless `--mouse-horizontal-scroll-speed-override` and/or `--mouse-vertical-scroll-speed-override` is also set?

Reply:
And what if you set both `-override`s, but not `-speed`? XD

My vote is on the above, but maybe with "x" as separator.
```
--mouse-scroll-speed {<hvscale>|<hscale>x<vscale>}
# e.g.
--mouse-scroll-speed 1.2
--mouse-scroll-speed -2
--mouse-scroll-speed 1x-1
--mouse-scroll-speed -1.2x2
```

Reply:
"But then… what if you only need vscroll changed?"

Reply:
Still nicer IMO than the `-vertical-override` option…
```
--mouse-scroll-speed 1x2
# or, but I don't think so, certainly not with `x` as separator
--mouse-scroll-speed x2
--mouse-scroll-speed 2x
```

Actually, there's one thing that needs consideration - in theory devices could have different defaults? So it may be important to have a way to only specify one axis? Or will the default always be `1.0`?